### PR TITLE
fix: fixed router history issue

### LINKED ISSUE #896: Investigate an issue with the history
When switching the route and then changing the locale, going back does not work properly. We need to investigate and adjust.

### PR DESCRIPTION
**Pull request by birtony**
Fixes #896

https://user-images.githubusercontent.com/33902374/128535916-cac7c591-669e-402f-98f8-486eb7fce7eb.mov

Signed-off-by: Anton Biriukov <anton.biriukov@securekey.com>

### CODE REVIEW

File: cmd/wallet-web/src/router/index.js [hunk 1]
```diff
@@ -46,30 +45,32 @@ router.beforeEach((to, from, next) => {
...
     if (store.dispatch('loadUser') && store.getters.getCurrentUser) {
       next();
       return;
     } else {
-      console.log('other options ');
-      next({
+      router.replace({
         name: 'block-no-auth',
-        params: { loginURL: `${locale.base}/signup` },
+        params: { locale: locale.base, loginURL: 'signup' },
```
Comment (talwinder50) [inline cmd/wallet-web/src/router/index.js:62]: SignupUrl ?

**Review by talwinder50 — CHANGES_REQUESTED** [review]

**Review by sudeshrshetty — APPROVED** [review]
Looks good to me except comment by @talwinder50

**Review by talwinder50 — APPROVED** [review]

**Review by sudeshrshetty — CHANGES_REQUESTED** [review]
lets confirm component reload while navigating back.

**Comment by birtony** [comment]
> lets confirm component reload while navigating back.

I have checked the flow, all resources load fine. I have also found [this issue](https://github.com/vuejs/vue-router/issues/897) on Vue Router's repo, which reflects our problem. More specifically, see https://github.com/vuejs/vue-router/issues/897#issuecomment-263052470. If we want to avoid having to go back twice, we need to use replace. @sudeshrshetty

**Review by sudeshrshetty — APPROVED** [review]

**Pull request merged**